app/src/main/cpp/Assertions.h

```diff
@@ -1,9 +1,9 @@
-#ifndef ASSERTIONS_H
-#define ASSERTIONS_H
+#pragma once
 
 // MACROS to define to-string and logging function utils
 #define STRINGIFY(x) #x
-#define FILE_AND_LINE __FILE__ ":" STRINGIFY(__LINE__)
```

Comment:
Isn't the FILE_AND_LINE macro used along the code ?  At least in OpenXRHelpers.h it is.

Reply:
Nevermind, I got it now.